.gitignore

```diff
@@ -27,6 +27,7 @@ yarn-error.log*
 
 # local env files
 .env*.local
+.env*.development
```

Comment:
Not sure why we are introducing another .env instance.  Can't local be treated as development..?